src/webextension/survey/components/fields/details.jsx

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react'
```

Comment:
It should now be linting JSX files with the `plugin:react/recommended` rules preset.